### ISSUE TITLE
Where can I find a good publish version with V4 support?

### ISSUE THREAD

**Issue by WilcoHei** [timols/java-gitlab-api#246]
Hey,

I am looking for a publish jar of java-gitlab-api version with v4 support.
In the following link I found a version with v4 but it seems like its missing all the functions.
http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22java-gitlab-api%22
I cannot find functions like createUsers and createGroup.
Can someone point my in the right direction?

**Comment by tomasbjerre** [comment]
See https://github.com/timols/java-gitlab-api/issues/222

**Comment by WilcoHei** [comment]
Are you saying that there is not a good version yet?

**Comment by EmilienMottet** [comment]
you can use [jitpack](https://jitpack.io/)

a stackoverflow answer https://stackoverflow.com/a/28483461/8362635

add in your pom.xml

``` xml
...
	<repositories>
		<repository>
		    <id>jitpack.io</id>
		    <url>https://jitpack.io</url>
		</repository>
	</repositories>

...
	<dependency>
	    <groupId>com.github.timols</groupId>
	    <artifactId>java-gitlab-api</artifactId>
	    <version>1.2.8</version>
	</dependency>
...

```

**Comment by WilcoHei** [comment]
Thx @EmilienMottet this is exactly what I am looking for.

**Comment by nafg** [comment]
#222 is closed, should this be too?

**Comment by timols** [comment]
Released v4.1.0 today
